tests/dragonfly/instance.py

```diff
@@ -169,8 +169,7 @@ def stop(self, kill=False):
                 proc.kill()
             else:
                 proc.terminate()
-            if proc.wait(timeout=15) < 0 and not kill:
```

Comment:
@kostasrim checking proc.wait(timeout=15) < 0 was a fix I made to check the process return code as if the process crashed on shutdown we did not failed the pytests. with this revert we have the same issue.
please provive a fix for this, I think you can check proc.returncode